interfaces/acados_template/acados_template/acados_ocp_solver.py

```diff
@@ -965,6 +965,8 @@ def get_stats(self, field_: str) -> Union[int, float, np.ndarray]:
             - time_solution_sens_lin: CPU time for linearization in eval_param_sens
             - time_solution_sens_solve: CPU time for solving in eval_solution_sensitivity
             - time_reg: CPU time regularization
+            - time_preparation: CPU time for last preparation phase, relevant for (AS-)RTI
+            - time_feedback: CPU time for last feedback phase, relevant for (AS-)RTI
```

Comment:
maybe add a comment that for non-RTI solvers time_preparation will return 0 and time_feedback will return time_tot?